PW33_2020_GranCanaria/Projects/NIRSOpenIGTLink/README.md

```diff
@@ -0,0 +1,46 @@
+Back to [Projects List](../../README.md#ProjectsList)
+
+# Write full project title here
+
+## Key Investigators
+
+- David Díaz Martín (ULL - MACbioIDi)
```

Comment:
You can probably add @Sunderlandkyl, he will be able to give advice on how to implement OpenIGTLink commands